tests/test_mlflow_lazily_imports_ml_packages.py

```diff
@@ -1,23 +1,14 @@
-# If test test script is run after other test script that imports ML packages in the same pytest
-# session, this test script fails. As a workaround, this test script needs to be run in a separate
-# pytest session with the following command:
-# $ pytest tests/test_mlflow_lazily_imports_ml_packages.py --lazy-import
 import sys
 
 import pytest
 
 
-def _get_loaded_packages():
-    return {k.split(".")[0] for k in sys.modules.keys()}
-
-
-# Run this test only when the `--lazy-import` flag is specified
-@pytest.mark.lazy_import
-def test_mlflow_lazily_import_ml_packages():
-    ml_packages = {
+@pytest.mark.parametrize(
+    "package",
+    (
         "catboost",
         "fastai",
-        "gluon",
+        "mxnet",
```

Comment:
why "gluon" removed ?

Reply:
See this line: https://github.com/mlflow/mlflow/blob/0bc7d2097270942d083c1168a1990c9091cb7ce4/mlflow/gluon/__init__.py#L76

Reply:
See this line: https://github.com/mlflow/mlflow/blob/0bc7d2097270942d083c1168a1990c9091cb7ce4/mlflow/gluon/__init__.py#L76

Reply:
`gluon` is a module in `mxnet`.